index.html

```diff
@@ -1,12 +1,144 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
-  </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
-
-  </body>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Седона</title>
+</head>
+<body>
+<header class="page-header" data-test="header">
+  <a class="header-logo">
+    <img src="#" alt="Логотип Sedona.">
+  </a>
+  <nav class="navigation">
+    <ul class="navigation-list">
+      <li class="navigation-item">
+        <a class="navigation-link navigation-link-current">Главная</a>
+      </li>
+      <li class="navigation-item">
+        <a class="navigation-link" href="#">О Седоне</a>
+      </li>
+      <li class="navigation-item">
+        <a class="navigation-link" href="hotels-catalog.html">Гостиницы</a>
+      </li>
+    </ul>
+    <ul class="navigation-list navigation-user">
+      <li class="navigation-item">
+        <a class="navigation-link" href="#">
+          <img src="#" alt="Поиск.">
+        </a>
+      </li>
+      <li class="navigation-item">
+        <a class="navigation-link" href="#">
+          <img src="#" alt="Избранное.">
+        </a>
+      </li>
+    </ul>
+    <button class="button" type="button">ХОЧУ СЮДА!</button>
+  </nav>
+</header>
+<main class="main-index" data-test="main">
+  <section class="hero" data-test="hero">
+    <h1 class="visually-hidden">Сайт города Седона</h1>
+    <img src="#" alt="Красивый вид на горы Седоны.">
```

Comment:
Здесь будет изображение не гор а картинка с Welcome, годы будем вставлять через css 

```html 
<section class="hero">
    <h1 class="visually-hidden">Главная страница о туризме в городе Седона</h1>
    <img src="./images/welcome.svg" alt="Welocome to the gorgeous Sedona Because thr Grand Canyon sucks!">
</section>

```